README.md

```diff
@@ -1,28 +1,65 @@
 # README
+This includes a guide on how to run the application locally and how to deploy it to Heroku.
 
-Nothing
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
+* Ruby version: 2.7.2
+* Rails Version: 6.1.1
 
 * System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
+  * Ubuntu (Either with WSL, VM or with regular Ubuntu. At least 18.x.)
+  * Gems
+    * Pg
+    * Puma
+    * Webpacker
+    * Turbolinks
+    * Jbuilder
+    * Bcrypt
+    * Devise
+    * Omniauth
+    * Omniauth-google-oauth2
+    * Bootsnap
+    * simplecov
+    * byebug
+    * rubocop
+      * performance
+      * rails
+      * rspec
+    * Web console
+    * Listen
+    * Spring
+    * Brakeman
+    * Capybara
+    * Rspec-rails
+    * Selenium-webdriver
+    * Faker
+    * Webdrivers
+    * Tzinfo-data
+* How to run
+  * Basic start up 
+    1. Clone repository
+    2. In root folder, run: ```rails db:migrate``` and ```rails db:seed```
+    3. Run ```rails s``` to start the server
+  * How to create a user of any role
+    1. Log into the application via a google account
+    2. Run ```rails c```
+    3. In the rails console run ```user = User.first``` or ```user = User.find_by(full_name: "full-name-here")```
+    4. Run ```user.user_role = UserRole.find_by(name: "role-name-here")```. Roles include "User", "Officer", or "President"
+  * If you can't log in because google says that the address wasn't added to the oauth list, contact lcaptain47
 
 * How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
+  * The application uses Rspec for testing
+  * Run ```rspec .``` in the root directory to run the entire test suite
+  * Run ```rspec spec/path/to/test.rb``` to run a specific spec file
+  * Run ```bundle exec rubocop --auto-correct``` to run rubocop and have it autocorrect most errors
+  * Run ```bundle exec brakeman -o brakeman.txt``` to run brakeman and have it output it's log to a text file in the root directory
 
 * Deployment instructions
```

Comment:
Love the detail on the sections as well, this makes a tutorial good because it has a defined direction and makes it very user friendly.  Bonus points for including the links.